### DISCUSSION TITLE
Trail path without mount

### DISCUSSION THREAD

**Discussion by cah-kangkung** [LadyElyssa/LadyElyssaTacoTrails#13] [General]
Hello, thx for the amazing trails. I'm just wondering is there any option to disable/change trails that use mounts? I found this option but it seems like it didn't do anything

![image](https://github.com/LadyElyssa/LadyElyssaTacoTrails/assets/40398027/1c9c8c1d-885c-46de-8d2d-e0455abf04c2)

**Comment by LadyElyssa** [comment]
Thank you for your comment and the Mount option you have de-selected in the picture toggles the mount markers. There are currently no Map Completion trails that cater for being on foot as Tekkit's Guides already has that covered, is that something you would like me to include?

Are there some areas you are struggling with, perhaps due to not having a particular mount like the skyscale?
